controllers/controllers.go

```diff
@@ -106,6 +106,31 @@ func (ctrl *OAuthController) HandleTokenRequest(w http.ResponseWriter, r *http.R
 	return ctrl.token(w, ctrl.Service.OauthServer.GetTokenData(ti), nil)
 }
 
+func (ctrl *OAuthController) TokenIntrospectHandler(w http.ResponseWriter, r *http.Request) {
+	token := strings.TrimPrefix(r.Header.Get("Authorization"), "Bearer ")
+	tokenInfo, err := ctrl.Service.OauthServer.Manager.LoadAccessToken(r.Context(), token)
+	if err != nil {
+		logrus.Errorf("Something went wrong loading access token: %s, token %s, request %v", err.Error(), token, r)
```

Comment:
- There is no need for a Sentry notification here imo. The error can be caused by a token simply being expired or already refreshed.
- I would not even log the error here, but we do need to call the [tokenError](https://github.com/getAlby/oauth2server/blob/7816eb33d02820f6fae947935e58c8339c26c7c9/controllers/controllers.go#L66) method.  We log the error in the tokenError function (I've made a PR to improve logging a little bit there).
- Then we need to return otherwise we will cause a panic because tokenInfo will be nil.